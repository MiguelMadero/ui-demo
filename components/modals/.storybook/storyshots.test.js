import initStoryshots, { snapshotWithOptions } from '@storybook/addon-storyshots';
import 'jest-styled-components';

const createNodeMock = element => {
  return document.createElement(element.type);
};

window.cancelAnimationFrame = () => {};
initStoryshots({
  configPath: __dirname,
  test: snapshotWithOptions({
    createNodeMock,
  }),
});
