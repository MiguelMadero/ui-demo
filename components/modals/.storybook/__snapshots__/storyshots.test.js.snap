// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots Confirmation Modal basic 1`] = `
.c0 {
  box-sizing: border-box;
}

.c5 {
  margin: 0px;
}

.c4 {
  font-weight: 400;
}

.c8 {
  font-size: 24px;
  color: inherit;
}

.c7::after {
  display: inline-block;
  font-family: Material-Design-Iconic-Font;
  text-align: center;
  content: '\\f3ec';
  -webkit-animation: zmdi-spin 1.5s infinite linear;
  animation: zmdi-spin 1.5s infinite linear;
}

.c3 {
  visibility: visible;
}

.c6 {
  position: absolute;
  top: -2px;
  left: 0;
  display: inline-block;
  margin-left: calc(50% - 12px);
  padding-top: inherit;
  pointer-events: none;
  visibility: hidden;
}

.c2 {
  background-color: #D6DEE9;
  color: #53616E;
}

.c1 {
  box-sizing: border-box;
  position: relative;
  cursor: pointer;
  border: none;
  -webkit-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
  text-decoration: none;
  border-radius: 2px;
  outline: 0;
  line-height: 1.43;
  text-align: center;
  height: 40px;
  min-width: 40px;
  font-size: 14px;
  padding: 10px 16px;
}

.c1:disabled {
  cursor: not-allowed;
  opacity: 0.65;
}

.c1:focus:not(:disabled),
.c1:hover:not(:disabled) {
  color: #53616E;
  background-color: #C8D0DB;
}

.c1:active:not(:disabled) {
  color: #53616E;
  background-color: #BCC4CF;
}

.c1:focus:not(:disabled) {
  box-shadow: 0 0 0 2px rgba(18,52,102,0.1);
}

<div
  className="c0"
  is={null}
>
  <div />
  <button
    className="c1 c2"
    disabled={false}
    is={null}
    mode="normal"
    onClick={[Function]}
    s="medium"
    type="button"
  >
    <p
      className="c3 c4 c5"
      hide={false}
      is={null}
    >
      Show modal
    </p>
    <span
      className="c6 c7 c8"
      hide={true}
      icon="spinner"
      is={null}
      spin={true}
    />
  </button>
</div>
`;

exports[`Storyshots Confirmation Modal in redux form 1`] = `
.c0 {
  box-sizing: border-box;
}

.c5 {
  margin: 0px;
}

.c4 {
  font-weight: 400;
}

.c8 {
  font-size: 24px;
  color: inherit;
}

.c7::after {
  display: inline-block;
  font-family: Material-Design-Iconic-Font;
  text-align: center;
  content: '\\f3ec';
  -webkit-animation: zmdi-spin 1.5s infinite linear;
  animation: zmdi-spin 1.5s infinite linear;
}

.c3 {
  visibility: visible;
}

.c6 {
  position: absolute;
  top: -2px;
  left: 0;
  display: inline-block;
  margin-left: calc(50% - 12px);
  padding-top: inherit;
  pointer-events: none;
  visibility: hidden;
}

.c2 {
  background-color: #D6DEE9;
  color: #53616E;
}

.c1 {
  box-sizing: border-box;
  position: relative;
  cursor: pointer;
  border: none;
  -webkit-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
  text-decoration: none;
  border-radius: 2px;
  outline: 0;
  line-height: 1.43;
  text-align: center;
  height: 40px;
  min-width: 40px;
  font-size: 14px;
  padding: 10px 16px;
}

.c1:disabled {
  cursor: not-allowed;
  opacity: 0.65;
}

.c1:focus:not(:disabled),
.c1:hover:not(:disabled) {
  color: #53616E;
  background-color: #C8D0DB;
}

.c1:active:not(:disabled) {
  color: #53616E;
  background-color: #BCC4CF;
}

.c1:focus:not(:disabled) {
  box-shadow: 0 0 0 2px rgba(18,52,102,0.1);
}

<div
  className="c0"
  is={null}
>
  <form>
    <div />
    <button
      className="c1 c2"
      disabled={false}
      is={null}
      mode="normal"
      onClick={[Function]}
      s="medium"
      type="button"
    >
      <p
        className="c3 c4 c5"
        hide={false}
        is={null}
      >
        Show modal
      </p>
      <span
        className="c6 c7 c8"
        hide={true}
        icon="spinner"
        is={null}
        spin={true}
      />
    </button>
  </form>
</div>
`;

exports[`Storyshots Modal default 1`] = `
.c0 {
  box-sizing: border-box;
}

.c5 {
  margin: 0px;
}

.c4 {
  font-weight: 400;
}

.c8 {
  font-size: 24px;
  color: inherit;
}

.c7::after {
  display: inline-block;
  font-family: Material-Design-Iconic-Font;
  text-align: center;
  content: '\\f3ec';
  -webkit-animation: zmdi-spin 1.5s infinite linear;
  animation: zmdi-spin 1.5s infinite linear;
}

.c3 {
  visibility: visible;
}

.c6 {
  position: absolute;
  top: -2px;
  left: 0;
  display: inline-block;
  margin-left: calc(50% - 12px);
  padding-top: inherit;
  pointer-events: none;
  visibility: hidden;
}

.c2 {
  background-color: #D6DEE9;
  color: #53616E;
}

.c1 {
  box-sizing: border-box;
  position: relative;
  cursor: pointer;
  border: none;
  -webkit-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
  text-decoration: none;
  border-radius: 2px;
  outline: 0;
  line-height: 1.43;
  text-align: center;
  height: 40px;
  min-width: 40px;
  font-size: 14px;
  padding: 10px 16px;
}

.c1:disabled {
  cursor: not-allowed;
  opacity: 0.65;
}

.c1:focus:not(:disabled),
.c1:hover:not(:disabled) {
  color: #53616E;
  background-color: #C8D0DB;
}

.c1:active:not(:disabled) {
  color: #53616E;
  background-color: #BCC4CF;
}

.c1:focus:not(:disabled) {
  box-shadow: 0 0 0 2px rgba(18,52,102,0.1);
}

<div
  className="c0"
  is={null}
>
  <div />
  <button
    className="c1 c2"
    disabled={false}
    is={null}
    mode="normal"
    onClick={[Function]}
    s="medium"
    type="button"
  >
    <p
      className="c3 c4 c5"
      hide={false}
      is={null}
    >
      Show modal
    </p>
    <span
      className="c6 c7 c8"
      hide={true}
      icon="spinner"
      is={null}
      spin={true}
    />
  </button>
</div>
`;
